.travis.yml

```diff
@@ -1,17 +1,26 @@
 sudo: false
+cache: bundler
 language: ruby
 
 rvm:
-  - 2.0
-  - 2.1
-  - 2.2
-  - jruby
-  - rbx-2
+  - 2.0.0-p648
+  - 2.1.8
+  - 2.2.4
+  - 2.3.0
```

Comment:
>> @Onumis CI is not fetching the latest version, don't know why. For example, if you specify 2.1 it will build against 2.1.5.
 
True, but then it should still not be hardcoded. How about rather opening up an issue at travis?